core/common/src/main/java/alluxio/grpc/ServiceVersionClientServiceHandler.java

```diff
@@ -67,6 +67,9 @@ public void getServiceVersion(GetServiceVersionPRequest request,
       case BLOCK_MASTER_WORKER_SERVICE:
         serviceVersion = Constants.BLOCK_MASTER_WORKER_SERVICE_VERSION;
         break;
+      case BLOCK_WORKER_CLIENT_SERVICE:
```

Comment:
interestingly `FILE_SYSTEM_WORKER_WORKER_SERVICE` was never here...

Reply:
I didn't notice... In that case it doesn't matter what we do with that protobuf definition. As long as new versions of the worker clients start checking version, they won't work with servers running old versions.